tests/test_prereleases_modular.py

```
@@ -502,3 +503,111 @@ def test_find_latest_remote_prerelease_dir_returns_none_when_no_match(
+            "firmware-2.7.26.deadbee",
+            "firmware-2.7.25.oldhash",
+        ]
+        admitted = prerelease_manager.scan_prerelease_directories(dirs, self.STABLE)
+
+        assert "2.7.27.abc123" in admitted
+        assert "2.8.0.abcdef1" in admitted
+        assert "2.7.26.deadbee" not in admitted
+        assert "2.7.25.oldhash" not in admitted
+
+    def test_scan_admits_multiple_newer_bases(
+        self, prerelease_manager, version_manager
```

Ruff (0.15.20)

[warning] Unused method argument: `version_manager`

(ARG002)